lib/zookeeper.js

```diff
@@ -305,29 +311,29 @@ Zookeeper.prototype.deletePartitionOwnership = function (groupId, topic, partiti
 };
 
 Zookeeper.prototype.addPartitionOwnership = function (consumerId, groupId, topic, partition, cb) {
-    var path = '/consumers/' + groupId + '/owners/' + topic + '/' + partition,
+    var path = this.pathPrefix+'/consumers/' + groupId + '/owners/' + topic + '/' + partition,
         self = this;
 
     async.series([
             function (callback) {
                 self.client.create(
-                        '/consumers/' + groupId,
+                        this.pathPrefix+'/consumers/' + groupId,

```

Comment:
What is the `this` context here? Should this line, 328, and 336 be using `self` instead of `this`?